docs/GettingStarted.md

```diff
@@ -0,0 +1,76 @@
+# Getting started
+
+All you'll need is a [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html) and the Hail source code. To clone the [Hail repository](https://github.com/broadinstitute/hail) using [Git](https://git-scm.com/), run
+```
+$ git clone https://github.com/broadinstitute/hail.git
+$ cd hail
+```
+The following commands are relative to the `hail` directory.
+
+You can also download the source code directly from [here](https://github.com/broadinstitute/hail/archive/master.zip).
+
+## Building and running Hail
+
+Hail may be built to run locally or on a Spark cluster. Running locally is useful for getting started, analyzing or experimenting with small datasets, and Hail development.
+
+### Locally
+The single command
+```
+$ ./gradlew installDist
+```
+installs Hail at `build/install/hail`. The initial build takes time as the build tool [Gradle](https://gradle.org/) installs and then downloads all Hail dependencies. The executable is `build/install/hail/bin/hail` (to run using `hail` add this `bin` to your path).
```

Comment:
"installs and then downloads" sounds strange.  The gradle wrapper downloads gradle.  Again, too much detail.  I'd just drop "installs".